.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,42 @@
+name: Update WPT
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+
+jobs:
+  update-wpt:
+    name: Update WPT
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332  # v4.1.7
+        with:
+          persist-credentials: false
+      - name: Update WPT
+        run: |
+          rm -rf test/fixtures/wpt
+          mkdir test/fixtures/wpt
+      - name: Clone WPT Repository
+        uses: actions/checkout@v2
+        with:
+          repository: web-platform-tests/wpt
+          ref: ${{ github.sha }}
+          path: test/fixtures/wpt
+          sparse_checkout: common/ eventsource/ fetch/ interfaces/ mimesniff/ resources/ service-workers/ storage/ websockets/ xhr/ LICENSE.md
+      - name: Move WPT Files
+        run: |
+            rm -rf test/fixtures/wpt/.git
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@c5a7806660adbe173f04e3e038b0ccdcd758773c  # v6.1.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: actions/wpt-update
+          title: Update WPT
+          body: Automated update of the WPT
+          commit-message: 'test: update wpt test'
```

Comment:
```suggestion
      - name: Create Or Update Pull Request
        uses: gr2m/create-or-update-pull-request-action@86ec1766034c8173518f61d2075cc2a173fb8c97  # v1.9.4
        env:
          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
        with:
          author: Node.js GitHub Bot <github-bot@iojs.org>
          body: This is an automated update of the WPT to ${{ env.NEW_VERSION }}.
          branch: actions/update-wpt  # Custom branch *just* for this Action.
          commit-message: 'test: update WPT to ${{ env.NEW_VERSION }}'
          labels: test
          title: 'test: update WPT to ${{ env.NEW_VERSION }}'
          update-pull-request-title-and-body: true
```

Reply:
Create Or Update Pull Request: Shouldn't the recommendation only cover Create Pull Request?

Reply:
Create or update pull request will allow it to update the WPT files again and again

Reply:
thanks